File: VB/Controllers/WebDocumentViewerController.vb
Imports System.Web.Mvc
Imports DevExpress.Web.Mvc.Controllers

Namespace MVCViewerWidgetDemo.Controllers
    Public Class WebDocumentViewerController
        Inherits WebDocumentViewerApiController

        Public Function CustomInvoke() As ActionResult
            Dim result = MyBase.Invoke()
            Response.AppendHeader("Access-Control-Allow-Origin", "*")
            Return result
        End Function
    End Class
End Namespace

File: CS/Controllers/WebDocumentViewerController.cs
using System.Web.Mvc;
using DevExpress.Web.Mvc.Controllers;

namespace MVCViewerWidgetDemo.Controllers {
    public class WebDocumentViewerController : WebDocumentViewerApiController {
        public ActionResult CustomInvoke() {
            var result = base.Invoke();
            Response.AppendHeader("Access-Control-Allow-Origin", "*");
            return result;
        }
    }
}